docs/RANDOM_SEEDS.md

```diff
@@ -297,9 +273,9 @@ This primarily leaves two kind of systems in the cold:
    boot. Alternatively, consider implementing a solution similar to
    systemd-boot's random seed concept in your platform's boot loader.
 
-2. Virtualized environments that lack both virtio-rng and RDRAND, outside of
-   test environments. Tough luck. Talk to your hosting provider, and ask them
-   to fix this.
+2. Virtualized environments that lack both virtio-rng and `GRND_INSECURE`,
```

Comment:
Is this an actual thing? GRND_INSECURE being unavailable because of configuration/whatnot by the hypervisor provider?

Reply:
Not quite. This may have just been a weird edit and maybe I should remove the bullet point all together? I was thinking that `virtio-rng` might initialize the rng pretty fast, so you don't even need GRND_INSECURE, while GRND_INSECURE fills the needs of RDRAND.

Do you want me to just remove the bullet point?

Reply:
Actually, the original text is fine. I'll restore that chunk. RDRAND _is_ relevant here because it can seed the kernel RNG.

Reply:
What about:

```
2. Virtualized environments that lack virtio-rng and a kernel without `GRND_INSECURE`,
   outside of test environments. Tough luck. Talk to your hosting provider, and ask them to
   fix this, or switch your VM to a newer kernel that provides `GRND_INSECURE` (v5.6
   and newer).
```

Reply:
sure, that works too

Reply:
That's what I was thinking at first too, but I just reread the whole section, and it's actually a discussion of bootstrapping the RNG to be secure, so I think the mention of GRND_INSECURE doesn't fit here, whilst the mention of RDRAND does, because:

```c
	if (arch_init && trust_cpu && !crng_ready()) {
		crng_init = 2;
		pr_notice("crng init done (trusting CPU's manufacturer)\n");
	}
```